python_modules/libraries/dagster-components/dagster_components/impls/dbt_project.py

```diff
@@ -109,11 +109,14 @@ def _fn(context: AssetExecutionContext, dbt: DbtCliResource):
 
     @classmethod
     def generate_files(cls, params: DbtGenerateParams) -> Mapping[str, Any]:
+        cwd = os.getcwd()
         if params.project_path:
-            relative_path = os.path.relpath(params.project_path, start=os.getcwd())
+            relative_path = os.path.relpath(params.project_path, start=cwd)
         elif params.init:
             dbtRunner().invoke(["init"])
-            subpaths = list(Path(os.getcwd()).iterdir())
+            subpaths = [
+                path for path in Path(cwd).iterdir() if path.is_dir() and path.name != "logs"
```

Comment:
What's up with the "logs" special-casing?

Reply:
dbt generates a `logs` directory to store logs when executing any command (including dbt init). Realistically should delete that dir here instead, just ignored it for now